node/pkg/watchers/sui/watcher.go

```diff
@@ -220,26 +260,26 @@ func (e *Watcher) inspectBody(logger *zap.Logger, body SuiResult) error {
 
 	txHashEthFormat := eth_common.BytesToHash(txHashBytes)
 
-	seq, err := strconv.ParseUint(*fields.Sequence, 10, 64)
+	seq, err := strconv.ParseUint(string(sequence), 10, 64)
```

Comment:
Do you need to do `string(sequence)` since sequence is a string?

Reply:
Good catch.